packages/chamelon/chamelon.0.1.0/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+maintainer: [ "yomimono <maintenance@identity-function.com>" ]
+authors: [ "yomimono <maintenance@identity-function.com>" ]
+homepage: "https://github.com/yomimono/chamelon"
+bug-reports: "https://github.com/yomimono/chamelon/issues"
+dev-repo: "git+https://github.com/yomimono/chamelon.git"
+license: "ISC"
+synopsis: "Subset of littlefs filesystem fulfilling MirageOS KV"
+description: """
+Chamelon implements a subset of the littlefs filesystem,
+which was originally designed for microcontroller use.  It exposes
+an interface matching the Mirage_kv.RW module type and operates
+on top of a block device matching Mirage_block.S .
+
+It is extremely not POSIX."""
+
+build: [
+ [ "dune" "build" "-p" name "-j" jobs ]
+ [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.10.0"}
+  "dune" {>= "2.9.0"}
+  "chamelon-unix" {= version & with-test}
```

Comment:
This is a dependency cycle in the tests. Could the tests that need chamelon-unix be moved to chamelon-unix?
This should be done easily by adding
```
(package chamelon-unix)
```
when needed

Reply:
Somewhat related but also the tests fails on chamelon-unix:
```
#=== ERROR while compiling chamelon-unix.0.1.0 ================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.10.2 | pinned(https://github.com/yomimono/chamelon/releases/download/v0.1.0/chamelon-v0.1.0.tbz)
# path                 ~/.opam/4.10/.opam-switch/build/chamelon-unix.0.1.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune runtest -p chamelon-unix -j 31
# exit-code            1
# env-file             ~/.opam/log/chamelon-unix-1107-bfe974.env
# output-file          ~/.opam/log/chamelon-unix-1107-bfe974.out
### output ###
# File "mirage_test/dune", line 3, characters 13-37:
# 3 |   (libraries mirage-block-combinators mirage-block-unix mirage-clock-unix mirage-crypto-rng mirage-crypto-rng.lwt mirage-kv logs.cli logs.fmt lwt chamelon chamelon.kv fpath alcotest alcotest-lwt)
#                  ^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Library "mirage-block-combinators" not found.
```